aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -63,19 +69,40 @@ module AptosFramework::Token {
         amount: u64,
     }
 
+    /// create collection event with creator address and collection name
+    struct CreateCollectionEvent has drop, store {
+        creator: address,
+        collection_name: ASCII::String,
+    }
+
+    /// token creation event id of token created
+    struct CreateTokenEvent has drop, store {
+        id: TokenId,
+    }
+
+    /// token creation event id of token created
+    struct CreateNonFungibleTokenEvent has drop, store {
+        id: TokenId,
+    }
+
     //
     // Core data structures for creating and maintaining tokens
     //
 
-    /// A token creator may have many tokens and many collections of tokens
+    /// Represent token collection meta-data owned by creator
+    /// A creator can have multiple collections
```

Comment:
/// Represents collection and token meta-data for a creator.

I think "A creator can have multiple collections" should be obvious from the fact that collections is a table. If you feel the comment is important, maybe put this above the field.